templates/node/test/services/service.test.js.twig

```diff
@@ -0,0 +1,38 @@
+const Client = require("../../lib/client");
+const InputFile = require("../../lib/inputFile");
+const {{ service.name | caseUcfirst }} = require("../../lib/services/{{ service.name | caseCamel }}");
+
+const mockedAxios = require("axios");
+jest.mock('axios', () => jest.fn());
```

Comment:
We are migrating to use `import {fetch} from 'undici'` as part of SSR support changes. (see https://github.com/appwrite/sdk-generator/pull/725)

We will need to mock that instead